test/unit_Line.jl

```diff
@@ -72,9 +72,13 @@ for N in [Float64, Rational{Int}, Float32]
     # concrete linear map of a line
     L = Line(N[1, -1], N(0)) # x = y
     M = N[1 0; 0 0] # non-invertible matrix
-    @test_throws ArgumentError linear_map(M, L)
-    M = N[2 2; 0 1] # invertible matrix
-    @test linear_map(M, L) == Line(N[0.5, -2.0], N(0.0))
+    if N == Rational{Int}
+        @test linear_map(M, L) isa HPolyhedron{Rational{BigInt}}
+    elseif N == Float32 || N == Float64
+        @test linear_map(M, L) isa HPolyhedron{Float64}
+    end
+    #M = N[2 2; 0 1] # invertible matrix
+    #@test linear_map(M, L) == Line(N[0.5, -2.0], N(0.0)) #TODO: see _linear_map_hrep_helper in Line.jl
```

Comment:
Revisit this after fixing the `Line` case. I do not see why this would not work.

Reply:
Good, the test can be uncommented as it works now.